inc/DatabaseHandler.php

```diff
@@ -310,39 +308,26 @@ function getUserAnswersOnDraws(array $drawIds, array $userNames): array {
     return $stmt->fetchAll();
   }
 
-  function saveLastAnswerQuery(int $ownerId, int $drawId): void {
+  function saveLastAnswerQuery(int $drawId): void {
     $stmt = $this->conn->prepare(
-     'UPDATE nq_owner_stats
-      SET last_answer = NOW(),
-          last_question = CASE WHEN last_draw_id = :drawId THEN last_question
-                               ELSE NULL END,
-          last_draw_id = :drawId
-      WHERE id IN (
-        SELECT stats_id
-        FROM nq_owner
-        WHERE id = :ownerId
-      );');
-    $stmt->bindParam('ownerId', $ownerId);
-    $stmt->bindParam('drawId', $drawId);
+      'INSERT INTO nq_draw_stats (draw_id, last_answer)
+       VALUES (:drawId, NOW())
+       ON DUPLICATE KEY UPDATE last_answer = NOW();');
```

Comment:
Replace with UPDATE, resp. some NOT NULL columns should probably be nullable if this statement can insert a new row.